doc/source/development/maintaining.rst

```diff
@@ -0,0 +1,177 @@
+.. _maintaining:
+
+******************
+Pandas Maintenance
+******************
+
+This guide is for pandas' maintainers. It may also be interesting to contributors
+looking to understand the pandas development process and what steps are necessary
+to become a maintainer.
+
+The main contributing guide is available at :ref:`contributing`.
+
+Roles
+-----
+
+Pandas uses two levels of permissions: **triage** and **core** team members.
+
+Triage members can label and close issues and pull requests.
+
+Core team members can label and close issues and pull request, and can merge
+pull requests.
+
+GitHub publishes the full `list of permissions`_.
+
+Tasks
+-----
+
+Pandas is largely a volunteer project, so these tasks shouldn't be read as
+"expectations" of triage and maintainers. Rather, they're general descriptions
+of what it means to be a maintainer.
+
+* Triage newly filed issues (see :ref:`maintaining.triage`)
+* Review newly opened pull request
+* Respond to updates on existing issues and pull requests
+* Drive discussion and decisions on stalled issues and pull requests
```

Comment:
If we are going to list what it means to be a maintainer / core team member, there is IMO missing one: think about API design or organisational issues, provide input on discussions, .. 

That might overlap with the line above, but I find it a bit narrow (it's not only relevant for stalled issues, although in those case the "final call" should probably come from maintainers)